src/redmodding.ts

```diff
@@ -128,10 +129,18 @@ const autoDeployActionCondition: V2077ActionConditionFunc = (
   }
   const state = vortexApi.store.getState();
   const gameMode = selectors.activeGameId(state);
-  const tool = redModTool(state, gameMode);
+  const tool = detectREDmoddingDlc(state, gameMode);
   return isSupported(gameMode) && (tool !== undefined);
 };
 
+export const internalSettingsViewStuff: V2077SettingsView = {
+  type: ActionType.AutoRunDeploy,
+  id: ``,
+  actionOrCondition: autoDeployAction,
+  condition: autoDeployActionCondition,
+  // test: checkTest,
+};
+
```

Comment:
I know I added this, but the name leaves something to be desired. Also I am not sure it is accurate. It's more like this deals with the ToDo side of the settings, not the Settings View really. Don't bother changing it in this commit though. I will get it taken care of myself if a refactoring ticket. Just kinda noting out my thoughts as I go.

Reply:
I nixed all this for now - I think it looks like something we may want to reintroduce but I tried to leave only code in use ATM